addOns/automation/src/test/java/org/zaproxy/addon/automation/jobs/ActiveScanConfigJobUnitTest.java

```diff
@@ -296,6 +301,64 @@ void shouldVerifyAndApplyInputVectors() {
         verify(param).setTargetParamsEnabledRPC(40);
     }
 
+    @Test
+    void shouldVerifyAndApplyExcludes() throws Exception {
+        // Given
+        Session session = mock(Session.class);
+        Model model = mock(Model.class);
+        Model.setSingletonForTesting(model);
+        given(model.getSession()).willReturn(session);
+
+        AutomationProgress progress = new AutomationProgress();
+        String yamlStr = "excludePaths:\n" + "- http://www.example\\..*\n" + "- .*\\.abc\n";
+        Object data = new Yaml().load(yamlStr);
+
+        AutomationEnvironment env = mock(AutomationEnvironment.class);
+        env.create(session, progress);
+
+        job.setEnv(env);
+        job.setJobData(((LinkedHashMap<?, ?>) data));
+
+        // When
+        job.verifyParameters(progress);
+        job.runJob(env, progress);
+
+        // Then
+        assertThat(progress.hasWarnings(), is(equalTo(false)));
+        assertThat(progress.hasErrors(), is(equalTo(false)));
+
+        @SuppressWarnings("unchecked")
+        ArgumentCaptor<List<String>> result = ArgumentCaptor.forClass(List.class);
```

Comment:
Use `captor()` which does not require the suppression.

Reply:
It does for me :/
```
> Compilation failed; see the compiler output below.
  /Users/simon/eclipse-workspace/zap-extensions/addOns/automation/src/test/java/org/zaproxy/addon/automation/jobs/ActiveScanConfigJobUnitTest.java:336: warning: [unchecked] unchecked method invocation: method forClass in class ArgumentCaptor is applied to given types
          ArgumentCaptor<List<String>> result = ArgumentCaptor.forClass(List.class);
                                                                       ^
    required: Class<S>
    found:    Class<List>
    where S,U are type-variables:
      S extends U declared in method <U,S>forClass(Class<S>)
      U extends Object declared in method <U,S>forClass(Class<S>)
  1 error
  2 warnings

```